__tests__/apiRequests.test.js

```diff
@@ -210,21 +210,66 @@ describe('APIs', () => {
         .get('/api/reviews')
         .expect(200)
         .then(({ body }) => {
-          const {reviewsArr }= body;
+          const { reviewsArr } = body;
           expect(reviewsArr).toHaveLength(13);
-          expect(reviewsArr).toBeSorted({descending: true})
-            expect(reviewsArr[4]).toEqual(
-              expect.objectContaining({
-                review_id: 3,
-                title: 'Ultimate Werewolf',
-                category: 'social deduction',
-                owner: 'bainesface',
-                review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
-                created_at: '2021-01-18T10:01:41.251Z',
-                votes: 5,
-                comment_count: 3,
-              })
-            );
+          expect(reviewsArr).toBeSorted({ descending: true });
+          expect(reviewsArr[4]).toEqual(
+            expect.objectContaining({
+              review_id: 3,
+              title: 'Ultimate Werewolf',
+              category: 'social deduction',
+              owner: 'bainesface',
+              review_img_url:
+                'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+              created_at: '2021-01-18T10:01:41.251Z',
+              votes: 5,
+              comment_count: 3,
+            })
+          );
+        });
+    });
+  });
+  describe.only('9. /api/reviews/:review_id/comments responds with', () => {
+    test('status 200 OK , returns an array of comments of the given review_id', () => {
+      return request(app)
+        .get('/api/reviews/3/comments')
+        .expect(200)
+        .then(({ body: { commentsArray } }) => {
+          expect(commentsArray.length).toBe(3);
+          expect(commentsArray[0]).toEqual(
+            expect.objectContaining({
+              comment_id: 2,
+              body: 'My dog loved this game too!',
+              review_id: 3,
+              author: 'mallionaire',
+              votes: 13,
+              created_at: '2021-01-18T10:09:05.410Z',
+            })
+          );
+        });
+    });
+    test('status 200 OK, valid review_id but no comments with that refference', () => {
```

Comment:
Id like you to instead respond with an empty array. We need consistency in our codebase. If we are expecting to render these comments out by looping through the array, if the array is empty nothing happens and we just dont see any. If we try to loop over this string that you are sending, or if you dont include a key with an empty array, we could be trying to loop over undefined which can cause errors in our front end